pkg/operator/ceph/pool/controller.go

```diff
@@ -152,6 +154,23 @@ func add(opManagerContext context.Context, mgr manager.Manager, r reconcile.Reco
 		return err
 	}
 
+	// Watch for updates to the secret triggered by changes in the peer pool token
+	err = c.Watch(
+		source.Kind(
+			mgr.GetCache(),
+			&corev1.Secret{TypeMeta: metav1.TypeMeta{Kind: "Secret", APIVersion: corev1.SchemeGroupVersion.String()}},
+			handler.TypedEnqueueRequestForOwner[*corev1.Secret](
```

Comment:
Is EnqueueRequestForOwner the right thing here? IIRC, the owner of the secret is the CephCluster, not the CephBlockPool right?